lib/src/metta/runner/stdlib/mod.rs

```diff
@@ -1,1835 +1,102 @@
 #[macro_use]
 pub mod math;
+pub mod random;
+pub mod atom;
+pub mod module;
+pub mod package;
```

Comment:
Package module is enabled only when `pkg_mgmt` feature is on
```suggestion
#[cfg(feature = "pkg_mgmt")]
pub mod package;
```